### ISSUE TITLE
Please write some Basic Documentation

### ISSUE THREAD

**Issue by hanumanjiblog** [flutter-webrtc/flutter-webrtc#640]
Please write some Basic Documentation, because please stop developing new features for few days and write some basic documentation otherwise that increase your GitHub issue.

**Comment by malaxmimdkm** [comment]
Yes, Add basic docs, documentation is more important than new features.

**Comment by cloudwebrtc** [comment]
@hanumanjiblog @malaxmimdkm I hope to add documents to this repo within two months, but now I need to find a way to solve my Household expenses first (since covid-19, I have to face the difficulties caused by the economic downturn), As you can see from the commit record, the number of code submissions to open source projects is declining.

flutter-webrtc is essentially a package of w3c webrtc API, so the usage of API is almost the same. If any of you have relevant development experience on webrtc, welcome to submit a PR about docs, and I can help review the docs PR.

**Comment by kingrupam** [comment]
@cloudwebrtc We understand your situation, when you free Please write 
Great Easy to Understand Documentation.
Thank You
